ui.frontend/src/view/FormFieldBase.js

```diff
@@ -604,6 +604,11 @@ class FormFieldBase extends FormField {
                 this.toggleAttribute(this.label, label.visible, Constants.ARIA_HIDDEN, true);
                 this.label.setAttribute(Constants.DATA_ATTRIBUTE_VISIBLE, label.visible);
             }
+
+            const bemClass = Array.from(this.element.classList).filter(bemClass => !bemClass.includes('--'))[0];
+            if (bemClass && this.label.parentElement.classList.contains(`${bemClass}__label-container`)) {
+                this.label.setAttribute(Constants.ROLE, "heading");
```

Comment:
role="heading" is not required on a label container. Labels are already accessible using the for attribute and the corresponding id.